modules/aspect_bazel_lib/0.11.4/MODULE.bazel

```diff
@@ -16,4 +12,6 @@ bazel_dep(name = "stardoc", version = "0.5.0", repo_name = "io_bazel_stardoc")
 
 ext = use_extension("@aspect_bazel_lib//lib:extensions.bzl", "ext")
 use_repo(ext, "jq_toolchains")
-use_repo(ext, "yq_toolchains")
\ No newline at end of file
+use_repo(ext, "yq_toolchains")
+register_toolchains("@jq_toolchains//:all")
```

Comment:
With the new API, I would find it a bit easier to read if the `register_toolchains` calls were coming right after the corresponding `use_repo`.

Reply:
Will fix that now!